documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -112,7 +114,9 @@ You can migrate virtual machines to {virt} from the command line.
 
 [IMPORTANT]
 ====
-You must ensure that all xref:prerequisites[prerequisites] are met.
+*  You must be logged in as a user with `cluster-admin` privileges. VMware only: The xref:vmware-privileges_{context}[VMware privileges] are sufficient.
```

Comment:
1. Please mention that the user must be logged in with Openshift/OKD `cluster-admin` privileges
2. I think it is better to move the 'VMware only note' a bit below, where the VDDK image is mentioned

Reply:
@mguetta1 I can move 'VMware only' there but without the cross-reference, so the user will not have any explanation on what the required privileges are.

Reply:
@sbeskin-redhat I understand. I think it can stay here just please make it a third point

Reply:
It is not a part of the first bullet. The first bullet is about **Openshift** privileges (should be cluster-admin) and this is why I suggested emphasizing that it is related to Openshift/OKD cluster
And the minimum set of privileges is about VMware user privileges